src/Hedgehog/Property.fs

```diff
@@ -128,6 +128,9 @@ module private Pretty =
     let private append (sb : StringBuilder) (msg : string) : unit =
         sb.AppendLine msg |> ignore
 
+    let private appendf (sb : StringBuilder) fmt =
+        Printf.kprintf (ignore << sb.AppendLine) fmt
```

Comment:
```suggestion
        Printf.ksprintf (ignore << sb.AppendLine) fmt
```

You meant [`ksprintf`](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-printfmodule.html#ksprintf), right? :arrow_up:

Reply:
Yeah, sorry. Good catch!